exercises/Novo-objeto.js

```diff
@@ -0,0 +1,108 @@
+/*  
+  Exercício 1: Criação e Acesso a Propriedades
+  Crie um objeto chamado carro com as seguintes propriedades:
+
+  marca (string)
+  modelo (string)
+  ano (número)
+*/
+const carro = {
+  marca: 'porsche',
+  modelo: '992.2',
+  ano: 2025,
+}
+console.log(carro);
+/* 
+  Exercício 2: Adicionar e Remover Propriedades
+  Dado o objeto abaixo:
+
+  Adicione uma propriedade chamada profissao com o valor 'Engenheira'.
+  Remova a propriedade idade.
+  Exiba o objeto atualizado no console.
+*/
+const pessoa = {
+  nome: 'Ana',
+  profissao: 'engenheira',
+};
+console.log(pessoa);
+
+
+/*
+  Exercício 3: Métodos em Objetos
+  Crie um objeto chamado calculadora com os seguintes métodos:
+
+  somar(a, b) - retorna a soma de dois números.
+  multiplicar(a, b) - retorna o produto de dois números.
+  Teste os métodos chamando-os com diferentes valores e exibindo o resultado no console.
+*/
+const multiplicar = (a, b) => a * b;
+console.log (multiplicar(3, 4));
+
+/*
+  Exercício 5: Desestruturação de Objetos
+  Dado o objeto:
+
+  Use desestruturação para extrair as propriedades nome e preco do objeto produto.
+  Extraia também o nome do fabricante e o país de fabricação do objeto aninhado fabricante.
+  Exiba os valores extraídos no console.
+*/
+const produto = {
+  nome: 'Notebook',
+  preco: 2500,
+  fabricante: {
+    nome: 'TechCorp',
+    pais: 'Brasil'
+  }
+};
+const {nome, preco, fabricante} = produto
```

Comment:
Boa, esqueci de te mostrar desestruturação aninhada, mas ficaria assim:
(ps: não precisa adicionar, vai causar um erro por causa do nome duplicado)

```suggestion
const { nome, preco, fabricante: { nome, pais } } = produto
```